src/app/modules/auth/sign-up/sign-up.component.html

```diff
@@ -14,120 +13,222 @@
-                    <div *ngIf="email.errors.email">{{'Email must be a valid email address' | translate}}</div>
-                  </div>
+              <div
+                *ngIf="fname.invalid && (fname.touched || signupForm.submitted)"
+                class="error-msg"
+              >
+                {{ 'First name is Required' | translate }}
               </div>
+            </div>
 
-              <div class="input-box required-field">
-                  <label for="password">{{'Password' | translate}}</label>
-                  <div class="input-box__border">
-                      <input placeholder="At least 8 characters" type="password" id="password" #password="ngModel" name="password" [(ngModel)]="signupObject.password" required />
-                  </div>
-                  <div *ngIf="password.invalid && (password.touched || signupForm.submitted)" class="error-msg">{{'Password is Required' | translate}}</div>
+            <div class="input-box required-field">
+              <label for="efname">{{ 'Email' | translate }}</label>
+              <div class="input-box__border">
+                <input
+                  placeholder="Email"
+                  type="email"
+                  id="email"
+                  #email="ngModel"
+                  name="email"
+                  [(ngModel)]="signupObject.email"
+                  required
+                  email
+                />
               </div>
+              <div
+                *ngIf="email.invalid && (email.touched || signupForm.submitted)"
+                class="error-msg"
+              >
+                <div *ngIf="email.errors.required">
+                  {{ 'Email is Required' | translate }}
+                </div>
+                <div *ngIf="email.errors.email">
+                  {{ 'Email must be a valid email address' | translate }}
+                </div>
+              </div>
+            </div>
 
-              <div class="input-checbox">
-                  <input class="cm-checkbox" type="checkbox" #agreedForTermsCondition="ngModel" id="agreedForTermsCondition"  name="agreedForTermsCondition" [(ngModel)]="signupObject.agreedForTermsCondition" required />
-                  <label for="agreedForTermsCondition">
-                    {{"I'm good with the Civis"| translate}}<br>
-                    <a href="/terms" class="policy-link">{{'Terms of Service,' | translate}}</a>
-                    &nbsp;
-                    <a href="/privacy" class="policy-link">{{'Privacy Policy' | translate}}</a>
-                    &
-                    <a href="/content-policy" class="policy-link">{{'Content Policy.' | translate}}</a>
-                  </label>
+            <div class="input-box required-field">
+              <label for="password">{{ 'Password' | translate }}</label>
+              <div class="input-box__border">
+                <input
+                  placeholder="At least 8 characters"
```

Comment:
@Harshwardhan18 Is it only atleast 8 characters? or `At least 8 characters with one special character and one numeric character`

Reply:
@Manzurkds Bro it's a placeholder. I don't think adding a whole guideline here's a good idea.
Can we do something like `*` then tell password requirements?

Reply:
@Harshwardhan18 Makes sense, you can just keep it as it is as this will be taken care in the invalid password message